### DISCUSSION TITLE
feature request - benchmark with https://github.com/RightTyper/RightTyper

### DISCUSSION THREAD

**Discussion by jaanaltosaar** [beartype/beartype#418] [Ideas]
would that be possible?

**Comment by leycec** [comment]
*...heh.* Rando new type hint inference package, huh? Looks sweet! @RightTyper definitely seems adjacent to @beartype's upcoming [type hint inference API](https://github.com/beartype/beartype/issues/403). Apparently, two bright minds both had the simultaneous bright idea to infer tensor type hints. <sup>*...curse ye, @RightTyper. curse ye to heck!*</sup>

That said, I'm kinda unsure what you'd like @beartype to do here. @RightTyper doesn't appear to be a *benchmarking* suite; it's a *type hint inference suite* from everything I can tell. Let's transition this into a discussion, maybe? Please regale me with your beautiful ideas, @jaanli. :smile:
